src/main/java/gregtech/common/metatileentities/multi/multiblockpart/appeng/MetaTileEntityMEInputHatch.java

```diff
@@ -245,11 +242,13 @@ public void registerAbilities(List<IFluidTank> list) {
         list.addAll(Arrays.asList(this.aeFluidTanks));
     }
 
-    public static class ExportOnlyAEFluid extends ExportOnlyAESlot<IAEFluidStack> implements IFluidTank, IFluidHandler, INotifiableHandler {
```

Comment:
removing `IFluidHandler` here means that this fluid tank can't be added to notifiable input fluids, since it has an instanceof check for `IFluidHandler`